### PR TITLE
Add Files module

### PR DESCRIPTION
**Pull request by LakshanSS**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/files/samples/copyFile.bal [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import ballerina/log;
+
+configurable string accessKeyOrSAS = ?; 
+configurable string accountName = ?;
+
+public function main() returns error? {
+    azure_files:AzureFileServiceConfiguration configuration = {
+        accessKeyOrSAS: accessKeyOrSAS,
+        accountName: accountName,
+        authorizationMethod : "accessKey"
```
Comment (SanduDS) [inline modules/files/samples/copyFile.bal:27]: Shall we give a ENUM value filtering here?

File: modules/files/tests/test.bal [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+import ballerina/os;
+import ballerina/test;
+
+configurable string accessKeyOrSAS = os:getEnv("ACCESS_KEY_OR_SAS");
+configurable string azureStorageAccountName = os:getEnv("ACCOUNT_NAME");
+
+AzureFileServiceConfiguration azureConfig = {
+    accessKeyOrSAS: accessKeyOrSAS,
+    accountName: azureStorageAccountName,
+    authorizationMethod : ACCESS_KEY
```
Comment (Chuhaa) [inline modules/files/tests/test.bal:27]: Can you check space?

Comment (LakshanSS) [reply modules/files/tests/test.bal]: Done

**Review by SanduDS — APPROVED** [review]

**Review by abeykoon — APPROVED** [review]

**Comment by LakshanSS** [comment]
Closing this PR since this is outdated (SL Alpha 2).
 Latest changes (in SL Alpha 4) are added in https://github.com/ballerina-platform/module-ballerinax-azure-storage-service/pull/5

**Pull request closed without merging**